tools/packaging/static-build/ovmf/build-ovmf.sh

```diff
@@ -22,14 +22,16 @@ package_output_dir="${package_output_dir:-}"
 DESTDIR=${DESTDIR:-${PWD}}
 PREFIX="${PREFIX:-/opt/kata}"
 architecture="${architecture:-X64}"
-if [ "${ovmf_build}" == "arm64" ]; then
+if [ "${ovmf_build}" == "arm64" ] || [ "${ovmf_build}" == "cca" ]; then
 	architecture="AARCH64"
 fi
 toolchain="${toolchain:-GCC5}"
 build_target="${build_target:-RELEASE}"
 
 [ -n "$ovmf_repo" ] || die "failed to get ovmf repo"
-[ -n "$ovmf_version" ] || die "failed to get ovmf version or commit"
+if [ -z "$ovmf_version" ] && [ -z "$ovmf_branch" ]; then
+    die "failed to get ovmf version or branch"
+fi
```

Comment:
I appreciate that we haven't solved all the existing issues in this file, but the new lines have the following shellcheck warnings:
```
./tools/packaging/static-build/ovmf/build-ovmf.sh:32:4: note: Prefer [[ ]] over [ ] for tests in Bash/Ksh/Busybox. [SC2292]
./tools/packaging/static-build/ovmf/build-ovmf.sh:32:10: note: Prefer putting braces around variable references even when not strictly required. [SC2250]
./tools/packaging/static-build/ovmf/build-ovmf.sh:32:30: note: Prefer [[ ]] over [ ] for tests in Bash/Ksh/Busybox. [SC2292]
./tools/packaging/static-build/ovmf/build-ovmf.sh:32:36: note: Prefer putting braces around variable references even when not strictly required. [SC2250]
```
so if you could correct these to not make the problems any worse, that would be great. Thanks